contrib/splitup.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -xe
+
+# VARIABLES
+BASEDIR=$(dirname "$0")
+LOCAL_BRANCH=$(cd $BASEDIR && git rev-parse --abbrev-ref HEAD)
+
+
+# FUNCTIONS
+function check_existing_remote {
+  if ! git remote | grep -sq $1; then
+    git remote add $1 git@github.com:/ceph/ansible-ceph-$1.git
+  fi
+}
+
+function pull_origin {
+  git pull origin --tags
+}
+
+function reset_hard_origin {
+  git checkout $LOCAL_BRANCH
+  git fetch origin
+  git reset --hard origin/master
+}
+
+# MAIN
+pull_origin
+
+for ROLE in $(ls -1 roles | grep -v ceph.ceph-common); do
```

Comment:
I'd suggest using ls -d -1 as you are expecting directories. That avoid making wrong stuff if someone drop a file here.